apollo-client/src/main/java/com/ctrip/framework/apollo/spring/annotation/EnableApolloConfig.java

```diff
@@ -69,4 +69,9 @@
    * @return
    */
   int order() default Ordered.LOWEST_PRECEDENCE;
+
+  /**
+   * Additional appId and namespace configurations. Will not participate in bootstrap
+   */
+  MultipleConfig[] multipleConfigs() default {};
```

Comment:
```suggestion

  /**
   * Additional appId and namespace configurations. Will not participate in bootstrap
   *
   * @since 2.4.0
   */
  MultipleConfig[] multipleConfigs() default {};
```